reactor-core/src/main/java/reactor/util/function/Tuple2.java

```diff
@@ -85,23 +85,37 @@ public Object get(int index) {
 	}
 
 	/**
-	 * Turn this {@literal Tuples} into a plain Object list.
+	 * Turn this {@code Tuple} into a {@link List List&lt;Object&gt;}.
+	 * The list isn't tied to this Tuple, unlike {@link #iterator()}, and therefore can
+	 * be used as one pleases, as a completely independent and <strong>mutable</strong> copy.
 	 *
-	 * @return A new Object list.
+	 * @return A mutable copy of the tuple as a new {@link List List&lt;Object&gt;}.
```

Comment:
`Arrays.asList` doesn't support `add` or `remove`, so mutability is limited to overwriting entries.

Reply:
@akarnokd oh snap, you're right 😆 
Well then, my argument in favor of keeping mutability is moot. But the good news is we have consistency between `toList()`, `toArray()` and `iterator()` 😁